### PR TITLE
PR: Add code style automatic checks on appveyor

### LINKED ISSUE #12: Add CI Integration

### LINKED ISSUE #15: Add automatic checks (ciocheck?)

### LINKED ISSUE #44: Fix enum34 if on circle ci

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #44 
Fixes #15 
Fixes #12 
 
@bemcdonnell this seems like a rather large PR, but most of the changes belong to the automatic code formatter (yapf and isort). Current style checks run for 2.7, 3.5 and 3.6 (although this is meant for running the actual tests, next PR)

I am using a library I helped write and we are using it on other projects (ciocheck)

Basically the CI tests will check if the code changes based on the autoformatter, if it does, then the test fails as it means you did not autoformat the code before pushing the commit. It will also check style for docstrings, pep8, flake8 and others!

This is also enforcing a 79 column limit, which we can change to 99 which is a more modern value.

So basically before making a commit you always need to run ciocheck

If using conda you can install it with

```
conda install ciocheck -c spyder-ide
ciocheck pyswmm -dt
```

This will run the checks but no tests. Tests will be updated in the next PR.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
+    - master 
 
 environment:
+  global:
+    # Used by qthelpers to close widgets after a defined time
+    TEST_CI: True
+    # Environment variables used by astropy helpers
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
```
Comment (bemcdonnell) [inline appveyor.yml]: SWMM5.dll is only supported in x86 right now thus the entire project is 32bit right now. Will this work?

**Comment by bemcdonnell** [comment]
Looking foreword to using this!

File: appveyor.yml (hunk 1 above)
Comment (goanpeca) [inline appveyor.yml]: Since we are not compiling stuff, for running tests it should just work

Comment (bemcdonnell) [inline appveyor.yml]: @goanpeca, maybe I'm missing something.  As i understand, a 64-bit library cannot talk to a 32-bit.  So if you are using 64-bit python, it cannot talk talk to the 32-bit SWMM dll. I've been wrong many times in my life so let me know if that is the case now ;)

Comment (goanpeca) [inline appveyor.yml]: its irrelevant for this PR, when the tests PR is enabled we can check that.

If you are right, not being able to run on 64 is a huge bummer :-\ if that is the case. Very few users use 32 bit python.

Comment (bemcdonnell) [inline appveyor.yml]: 64-bit swmm is available but remains untested.  Once we get CI for the OWA SWMM Repo and testing, we should have a better idea on how the results compare between 32 and 64 bit.  In the mean time, Go ahead and merge this PR

Comment (goanpeca) [inline appveyor.yml]: @michaeltryby, @bemcdonnell would you guy want me to add CI testing for the OWA SWMM?, based on similar work to what was done here?

Comment (bemcdonnell) [inline appveyor.yml]: @goanpeca Yes yes yes yes yes!
